dev/Sonos-Kids-Controller-master/package.json

```diff
@@ -7,7 +7,8 @@
         "ng": "ng",
         "start": "node server.js",
         "build": "ionic build --prod",
-        "deploy": "ionic build --prod"
+        "deploy": "ionic build --prod",
+        "test": "ng test"
```

Comment:
This change allows to run `npm test` to run the Angular frontend tests (to execute the `*.spec.ts` files).